app/Http/Controllers/Api/CtsController.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App\Http\Controllers\Api;
+
+use App\Repositories\Cts\BookingRepository;
+use Illuminate\Http\Request;
+
+class CtsController
+{
+    private $bookingRepository;
+
+    public function __construct(BookingRepository $bookingRepository)
+    {
+        $this->bookingRepository = $bookingRepository;
+    }
```

Comment:
nitpick: you might benefit from the new constructor initialisation syntax

```suggestion
    public function __construct(private BookingRepository $bookingRepository)
    {}
```